playwright/tests/organize/people/person/tags.spec.ts

```diff
@@ -357,16 +357,16 @@ test.describe('Person Profile Page Tags', () => {
         'New tag title'
       );
 
-      await page.click('data-testid=submit-button');
-
       // Check that request made to edit tag with correct values
       await Promise.all([
+        page.click('data-testid=submit-button'),
```

Comment:
For this to be 100% safe I believe the `waitForResponse()` needs to be invoked _before_ the click. That's absolutely not critical though, because I think it's extremely unlikely (or impossible?) that the click and it's request would finish _synchronously_ before moving to the next statement.